### PR TITLE
🚨🚨🐛 Source Coda: Update Schemas and Fix Tests

### PR DESCRIPTION
**Pull request by archangelic**
## What
Corrects fields and adds fields to schemas.

## How
*Describe the solution*

## Recommended reading order
1. `source-coda/schemas/docs.json`
2. `source-coda/schemas/pages.json`

## 🚨 User Impact 🚨
The docs schema had a field `owner_name` which has been renamed to `ownerName`. This is a breaking change. Pages schema has additional fields.

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- Connector version is set to `0.0.1`
    - `Dockerfile` has version `0.0.1`
- Documentation updated
    - Connector's `README.md`
    - Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - `docs/integrations/<source or destination>/<name>.md` including changelog with an entry for the initial version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - `docs/integrations/README.md`

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Connector Generator</strong></summary>

- Issue acceptance criteria met
- PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests#pull-request-title-convention)
- If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-coda test report (commit [`160f471a22`](https://github.com/airbytehq/airbyte/pull/28093/commits/160f471a228c18d7fd3d236440dc1c6514840c97)) - ❌

⏲️  Total pipeline duration: 02mn53s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-coda/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ❌        |
| Code format checks                                                 | ✅        |
| Connector package install                                          | ✅        |
| Build source-coda docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Acceptance tests                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/mal_fix-coda/1689015564/160f471a228c18d7fd3d236440dc1c6514840c97/source-coda/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-coda test
```

**Comment by octavia-squidington-iii** [comment]
##  source-coda test report (commit [`266d1591a4`](https://github.com/airbytehq/airbyte/pull/28093/commits/266d1591a4d5304034388c04c40917e84080f69f)) - ✅

⏲️  Total pipeline duration: 02mn15s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-coda/metadata.yaml | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |
| Code format checks                                                 | ✅        |
| Connector package install                                          | ✅        |
| Build source-coda docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                         | ✅        |
| Acceptance tests                                                   | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/mal_fix-coda/1689016265/266d1591a4d5304034388c04c40917e84080f69f/source-coda/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-coda test
```

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-coda/Dockerfile [hunk 1]
```diff
@@ -34,5 +34,5 @@ COPY source_coda ./source_coda
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.version=1.0.0
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-coda/Dockerfile:37]: Why a major update and not a minor one?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-coda/source_coda/schemas/pages.json [hunk 2]
```diff
@@ -159,6 +159,82 @@
...
+            "type": "string"
+          }
+        }
+      }
+    },
+    "updatedBy": {
+      "type": ["null", "object"],
+      "properties": {
+        "@context": {
+          "type": "string"
```
[the remark this answers is not in the document]
Comment (archangelic) [reply airbyte-integrations/connectors/source-coda/source_coda/schemas/pages.json:220]: Do you think it's best to do this to all the types?

File: airbyte-integrations/connectors/source-coda/Dockerfile (hunk 1 above)
Comment (archangelic) [reply airbyte-integrations/connectors/source-coda/Dockerfile:37]: The docs schema had a field `owner_name` which has been renamed to `ownerName`.

Comment (archangelic) [reply airbyte-integrations/connectors/source-coda/Dockerfile:37]: If I'm not mistaken, renaming a field is a major update.

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**